Gems/Atom/RHI/DX12/Code/Source/RHI/StreamingImagePool.cpp

```diff
@@ -461,7 +461,7 @@ namespace AZ
         {
             AZ_PROFILE_FUNCTION(RHI);
 
-            Image& image = static_cast<Image&>(*request.m_image);
+            auto& image = static_cast<Image&>(*request.m_image.get());
```

Comment:
When you get a reference like here, you don't have to call `get()`, since the `operator*` is overloaded and gives you the same reference, i.e., you can revert all the added `get()` calls in this PR when the target is a reference (doesn't work for pointers though).

Reply:
reverted.